submission/submission-config/src/main/kotlin/ac/uk/ebi/biostd/common/properties/ApplicationProperties.kt

```diff
@@ -12,6 +12,7 @@ data class ApplicationProperties(
     val requestFilesPath: String,
     val ftpPath: String,
     val instanceBaseUrl: String,
+    val baseSubmissionRelPath: String = "",
```

Comment:
just `subBasePath` sound better for me and we can add comment documenting , instead of making it an empty string lets make a nullable field.